recipes/slamdunk/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
please use pip install, have a look here: https://github.com/bioconda/bioconda-recipes/blob/31f2bc37fa14a7a1b51451383bcb39909f50f741/recipes/pymzml/0.7.5/meta.yaml#L12

Reply:
It won't work unfortunately. When I run this build command, it will just skip with

```
07:34:27 BIOCONDA INFO Nothing to be done for recipe recipes/slamdunk
```